src/Libraries/Microsoft.Extensions.AsyncState/FeaturesPooledPolicy.cs

```diff
@@ -1,20 +1,21 @@
 ﻿// Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+using System.Collections.Concurrent;
 using Microsoft.Extensions.ObjectPool;
 
 namespace Microsoft.Extensions.AsyncState;
 
-internal sealed class FeaturesPooledPolicy : IPooledObjectPolicy<Features>
+internal sealed class FeaturesPooledPolicy : IPooledObjectPolicy<ConcurrentDictionary<AsyncStateToken, object?>>
```

Comment:
nit: the name of the class might be updated since we don't have `Features` type anymore

Reply:
Done